src/Workspaces/Core/Portable/Workspace/Solution/ProjectState_Checksum.cs

```diff
@@ -5,6 +5,7 @@
 #nullable disable
 
 using System;
+using System.Collections.Immutable;
```

Comment:
nit: intentional?

Reply:
nit: How has a nullable disabled lasted this long in this file?

Reply:
Removed in followup.